content/blog/2021-12-21-December-21-community-gems.md

```diff
@@ -0,0 +1,218 @@
+---
+title: December '21 Community Gems
+date: 2021-12-21
+description: >
+  A roundup of technical Q&A's from the DVC community. This month: comparing
+  experiments, working with data, working with pipelines, and more.
+descriptionLong: >
+  A roundup of technical Q&A's from the DVC community. This month: comparing
+  experiments, working with data, working with pipelines, and more.
+picture: 2021-12-21/dec-community-gems.png
+author: milecia_mcgregor
+commentsUrl: https://discuss.dvc.org/t/december-21-community-gems/1001
+tags:
+  - Data Versioning
+  - DVC Remotes
+  - DVC API
+  - DVC Stages
+  - Community
+---
+
+### [While doing a `dvc pull`, I accidentally entered the verification from the wrong Google account. How can I edit that?](https://discord.com/channels/485586884165107732/563406153334128681/908437162150739978)
+
+No problem @fireballpoint1! This happens sometimes.
+
+You should be able to run the following command in your terminal and then
+re-enter your credentials.
+
+```dvc
+$ rm .dvc/tmp/gdrive-user-credentials.json
+```
+
+That should give you a chance to enter the correct credentials when you try to
+`dvc pull` again.
+
+### [Can I add a `dvc remote` which refers to NAS by IP so I don't have to mount on every computer?](https://discord.com/channels/485586884165107732/563406153334128681/912667503283564544)
+
+That's a new question for us @Krzysztof Begiedza!
+
+If you enable the SSH service on your NAS, you can configure DVC to use it as an
+SSH remote.
+
+There should also be DSM packages for webdav as well, if you prefer that over
+SSH. Just make sure that your URL looks similar to this.
+
+```
+webdav://<ip>/<path>
+```
+
+### [Can you selectively `dvc pull` data files?](https://discord.com/channels/485586884165107732/563406153334128681/913713923667148850)
+
+Great question @Clemens!
+
+You would `dvc import` the relevant files into each of your individual projects
```

Comment:
I don't quite understand this answer to be honest, why do I need to import anything? If I can do `dvc pull <file>` or pull has `--glob` option to specify what to pull.

In their case the could make a data registry (add link to use case) and then use `dvc import` in other projects to get a specific dataset (no need for granular pull in that case).

I would mention both of those option in that seauence.